### PR TITLE
feat: billing usage and cluster limit validation

### PR DESCRIPTION
**Pull request by CristhianF7**
### Cluster usage table

Screenshot 2024-01-09 at 1 20 23 PM

### Cluster limit modal

Screenshot 2024-01-09 at 12 30 11 PM

### CODE REVIEW

File: app/settings/subscription/[slug]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import axios from 'axios';
+import sortBy from 'lodash/sortBy';
+
+import Subscription from '@/containers/subscription';
+import { Plan } from '@/types/plan';
+
+async function getProductPlans() {
+  const { SAAS_API_URL = '' } = process.env;
+
+  const { data } = await axios.get<Array<Plan>>(`${SAAS_API_URL}/api/v1/payment/plans`);
```
Comment (jairoFernandez) [inline app/settings/subscription/[slug]/page.tsx:11]: WDYT: if you move these request to .../[slug]/route.ts to get advantage of server request

**Review by jairoFernandez — DISMISSED** [review]
🚀 just a few comments

File: app/settings/subscription/[slug]/page.tsx (hunk 1 above)
Comment (CristhianF7) [reply app/settings/subscription/[slug]/page.tsx:11]: this is already a server request

Comment (jairoFernandez) [reply app/settings/subscription/[slug]/page.tsx:11]: I guess is true in the previous version of nextjs with getServerSideProps o getStaticProps, but here looks like a client execution, but I am not 100% sure

**Pull request merged**